### ISSUE TITLE
softmax test run reports success on failure

### ISSUE THREAD

**Issue by tacotown** [udacity/dlnd-issue-reports#480]

**Comment by tacotown** [comment]
Trying for L=[5,6,7].
The correct answer is
[0.09003057317038046, 0.24472847105479764, 0.6652409557748219]
And your code returned
[0.003033106151596548, 0.0036397273819158576, 0.004246348612235167]

Correct!

**Comment by lcrucks** [comment]
Hello, could you report exactly where this is found?  (Section, Lesson, node number and name?)

**Comment by nzhang-zh** [comment]
Encountered same issue.
DLND -> 2. Neural Networks -> 1. Introduction to Neural Networks -> 15. Softmax -> Quiz

Simply doing the following passes the test.
```
import numpy as np

# Write a function that takes as input a list of numbers, and returns
# the list of values given by the softmax function.
def softmax(L):
    return L
```

> Trying for L=[5,6,7].
The correct answer is
[0.09003057317038046, 0.24472847105479764, 0.6652409557748219]
And your code returned
[5, 6, 7]
Correct!
